sources/packages/backend/apps/api/src/services/student/student.service.ts

```diff
@@ -166,11 +168,14 @@ export class StudentService extends RecordDataModelService<Student> {
     student.sinConsent = studentInfo.sinConsent;
     try {
       // Get PD status from SFAS integration data.
-      student.studentPDVerified = await this.sfasIndividualService.getPDStatus(
-        user.lastName,
-        student.birthDate,
-        studentSIN,
-      );
+      const sfasStudentPDVerified =
+        await this.sfasIndividualService.getPDStatus(
+          user.lastName,
+          student.birthDate,
+          studentSIN,
+        );
+      student.studentPDVerified = sfasStudentPDVerified;
+      student.disabilityStatus = this.getPDStatus(sfasStudentPDVerified);
```

Comment:
When the student is not present in the SFAS system, then also a PD Status will be returned, right? However, in that scenario we actually do not know the PD Status.

Reply:
The intention of the logic is, if the newly created student is already present is SFAS, then read the pd_status, otherwise create the student with default PD status which is "Not requested"